gulp-tasks/reference-docs/build-js-docs.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
```

Comment:
Add the JSDoc description to this file and include at least `@fileoverview` and `@author`

Reply:
Done.